### PR TITLE
Initial copy of session.cache_fat using sessionDatabase-1.0

### PR DESCRIPTION
**Pull request by aguibert**

### CODE REVIEW

**Comment by aguibert** [comment]
#build

File: dev/com.ibm.ws.session.cache_fat/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+bVersion=1.0
+
+src: \
+	fat/src,\
+	test-applications/sessionCacheApp/src
+
+fat.project: true
+
+-buildpath: \
+	com.ibm.websphere.javaee.servlet.3.1;version=latest,\
```
Comment (njr-11) [inline dev/com.ibm.ws.session.cache_fat/bnd.bnd]: use latest, servlet-4.0

File: dev/com.ibm.ws.session.cache_fat/publish/servers/sessionCacheServer/server.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<server>
+
+    <featureManager>
+        <feature>servlet-3.1</feature>
```
Comment (njr-11) [inline dev/com.ibm.ws.session.cache_fat/publish/servers/sessionCacheServer/server.xml]: Use latest:
servlet-4.0
and
jdbc-4.2

File: dev/com.ibm.ws.session.cache_fat/test-applications/sessionCacheApp/src/session/cache/web/SessionCacheTestServlet.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        Assert.assertNotNull("The dataSource was not found in the HTTP session", sessionDS);
+        sessionDS.getConnection().close();
+        sessionDS = (DataSource) sessionMap.get("dataSource");
+        Assert.assertNotNull("The indirect dataSource was not found in the HTTP session", sessionDS);
+        sessionDS.getConnection().close();
+    }
+
+    @Test
+    public void basicTest(HttpServletRequest request, HttpServletResponse response) throws Exception {
+        System.out.println("@AGG working");
```
Comment (njr-11) [inline dev/com.ibm.ws.session.cache_fat/test-applications/sessionCacheApp/src/session/cache/web/SessionCacheTestServlet.java]: Not harmful, but probably not an intended delivery

**Pull request merged**